app/assets/javascripts/app/form-validation.js

```diff
@@ -21,7 +20,6 @@ const validate = {
   },
 
   addEvents() {
-    this.form.addEventListener('change', e => validateField(e.target));
```

Comment:
Yeah we should keep this functionality - not having a test for it was an oversight on my part.